Architects/Technical Architect.md

```diff
@@ -0,0 +1,147 @@
+# Technical Architect
+
+## Department for Education
+
+## About the job
+
+### Summary
+
+These are exciting times at the Department for Education. We are transforming to become a user-centred organisation, delivering better outcomes for all our users.  
+  
+[This video](https://www.youtube.com/watch?v=dgS-yJKR4Z4) gives an insight into the culture which is being developed within some of the teams.  
+  
+As a Technical Architect, you will be designing and building services for a diverse set of users, including citizens, teachers, social workers, and school professionals.  
+  
+You will be working in [for example] the Schools Business Professionals digital delivery portfolio, which includes services like Get help buying for your school, Start and manage an academy and Build or repair a school.
+
+### Job description
+
+Your main responsibilities will include:
+
+  
+

```

Comment:
Looks like there are lots of extra carriage returns throughout - could we remove please?